NetcoinCLI/Menu.cs

```diff
@@ -82,45 +80,76 @@ public void HandleMenuSelection(string input)
                     case "skapa kund":
-                        Console.Write("\nFör in uppgifter för att skapa kund: \n\tNamn: ");
+                        Console.Write("\nFör in uppgifter för att skapa kund: \nNamn: ");
                         var name = Console.ReadLine();
-                        Console.Write("Personnummer: ");
+                        Console.Write("Organisationsnummer: ");
                         var legalId = Console.ReadLine();
                         Console.Write("Gatuadress: ");
                         var address = Console.ReadLine();
                         Console.Write("Postnummer: ");
                         var postalCode = Console.ReadLine();
+                        Console.Write("Stad: ");
+                        var city = Console.ReadLine();
                         Console.Write("Område: ");
                         var area = Console.ReadLine();
-                        //TODO Add a service to add a customer.
+                        Console.Write("Land: ");
+                        var country = Console.ReadLine();
+                        Console.Write("Telefonnummer: ");
+                        var phonenumber = Console.ReadLine();
+                        var isCreatedOrNot = _bankSystem.CreateCustomer(name,legalId,area,address,postalCode,city,country,phonenumber);
+                        if (isCreatedOrNot)
+                        {
+                            Console.Write($"Kund {name}, med organistaionsnummer: {legalId} skapad");
+                        }
+                        else
+                        {
+                            Console.Write("Något gick fel när du skapade kunden. Kontakta support.");
+                        }
                         break;
 
                     case "4":
                     case "ta bort kund":
                         Console.Write("\nSkriv in kundnummer för den du vill radera: ");
-                        customerId = int.Parse(Console.ReadLine());
-                        //TODO Add a service to delete a customer.
+                        int customerId = int.Parse(Console.ReadLine());
+                        if (_bankSystem.RemoveCustomer(customerId))
+                            Console.WriteLine($"Kund med ID { customerId } med tillhörande konton har tagits bort.");
+                        else
+                            Console.WriteLine("Kunden kunde inte tas bort - var god kontrollera kundnummer och att dess konton har 0 i saldo.");
                         break;
 
                     case "5":
                     case "skapa konto":
                         Console.Write("\nAnge kundnummer där du vill att nya kontot skapas: ");
-                        customerId = int.Parse(Console.ReadLine());
-                        //TODO Add a service to create a account.
+                        var accountCreatedOrNot = _bankSystem.CreateAccount(int.Parse(Console.ReadLine()));
+                        if (accountCreatedOrNot)
```

Comment:
Strangely named variable. What would the else-clause mean with this name? Suggestion: Remove the "OrNot"-part of the identifier, or name it something that communicates success.